packit/config.py

```diff
@@ -85,7 +85,21 @@ def __init__(self):
         self._pagure_user_token: str = ""
         self._pagure_fork_token: str = ""
         self.dry_run: bool = False
-        self.actions_handler: Optional[str] = "local"
+
+        # %%% ACTIONS HANDLER CONFIGURATION %%%
```

Comment:
Do you plan to add this section also into `https://github.com/packit-service/packit/blob/master/docs/configuration.md`?

Reply:
certainly not so we would not confuse packit tool users, we should add that into docs of service deployment